### PR TITLE
Don't remove empty ship folders on uninstall 

### PR DESCRIPTION
**Pull request by Postremus**
also check for severall other directories
which should not be removed

### CODE REVIEW

**Comment by Postremus** [comment]
fixes #1191

File: Core/ModuleInstaller.cs [hunk 1]
```diff
@@ -793,8 +793,12 @@ private void Uninstall(string modName)
                     if (!Directory.EnumerateFileSystemEntries(directory).Any())
                     {
                         // Skip Ships/VAB ans Ships/SPH
-                        if (directory == KSPPathUtils.ToAbsolute("VAB", ksp.Ships())
-                            || directory == KSPPathUtils.ToAbsolute("SPH", ksp.Ships()))
+                        string vab = ksp.ShipsVab();
+                        string sph = ksp.ShipsSph();
```
Comment (pjf) [inline Core/ModuleInstaller.cs]: These two strings never get used!

**Comment by pjf** [comment]
This is excellent. Any chance we can get `@thumbs` protected as well?

**Comment by Postremus** [comment]
@pjf done

**Pull request merged**